l10n_it_ddt/migrations/12.0.1.7.1/pre-migrate.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+_field_renames = [
+    ('stock.picking.package.preparation', 'stock_picking_package_preparation',
+     'show_price', 'ddt_show_price'),
+]
+
+
+@openupgrade.migrate()
+def migrate(env, version):
```

Comment:
È stato testato anche questo?

Reply:
Sì @eLBati.

```
2020-08-10 08:35:50,333 6252 INFO customer-db odoo.modules.migration: module l10n_it_ddt: Running migration [>12.0.1.7.1] pre-migrate
2020-08-10 08:35:50,459 6252 INFO customer-db OpenUpgrade: l10n_it_ddt: pre-migration script called with version 12.0.1.4.3
2020-08-10 08:35:50,461 6252 INFO customer-db OpenUpgrade: table stock_picking_package_preparation, column show_price: renaming to ddt_show_price
2020-08-10 08:35:50,476 6252 DEBUG customer-db OpenUpgrade: Deleting record for XML-ID l10n_it_ddt.ddt_stock_picking_package_preparation_form
```

Occorre però poter disporre dell'ultima versione di `openupgradelib` prelevata da master così come indicato in https://github.com/OCA/OpenUpgrade/commit/389c78d0383e6ee1605c5cde92931eba85a15baf altrimenti si
ricade nell'errore https://www.odoo-italia.org/forum/forum-1/question/odoo-12-aggiornamento-modulo-oca-contract-errore-nel-aggiornamento-1039

Reply:
@tafaRU grazie

Reply:
@eLBati puoi fare review?